### PR TITLE
New visualizations and color scheme changes

### PR DESCRIPTION
**Pull request by claban3**
This PR adds two new visualizations, and uses monochromatic gradients to spice up all the visuals. Please take a look at the PR build. I also temporarily expanded the animation toolbar so you could see these animations, but @aposp will remove this when his animation settings panel is implemented.

### CODE REVIEW

File: src/Types/Animations/WallBounce.tsx [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { SketchData } from '../Figures';
 
 export class WallBounce extends Animation {
     static draw(sketchData: SketchData, p) {
+        p.background(255);
+        p.fill(100);
```
Comment (cannono) [inline src/Types/Animations/WallBounce.tsx:6]: do sum about this

File: src/Views/Settings/Settings.css [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
 
-.settings-container.opening {
+/* .settings-container.opening {
     animation: slideUp .2s linear;
 }
 
 .settings-container.closing{
     animation: slideDown .2s linear;
-}
+} */
```
Comment (cannono) [inline src/Views/Settings/Settings.css:20]: don't kill my baby

File: src/Types/ProcessingFigures.tsx [hunk 3]
```diff
@@ -129,17 +136,34 @@ export class CircleFigure extends AnimatedFigure {
...
+      this.velocity.normalize().mult(MAX_SPEED);
+    }
+    customStyles.stroke ? this.p5.stroke(1) : this.p5.noStroke();
+    
+    if (customStyles.randomFill) {
+      let color = this.p5.color(this.color);
+      color.setAlpha((this.dim * 7));
+      this.p5.fill(color);
+    }
+
```
Comment (cannono) [inline src/Types/ProcessingFigures.tsx:157]: What do you need sketchdata for?

**Review by cannono — APPROVED** [review]

**Comment by aposp** [comment]
@claban3 Small bug I've noticed with the new `downwardGravity` - the shapes, as they shrink and fade out, leave very faint trails on the screen. This is reproducible by filling the bottom half of the screen with "no animation" shapes, then turning on the gravity animation so they fall. Clearing the canvas makes this contrast more obvious.

**Pull request merged**